Mathlib/Geometry/Euclidean/Angle/Unoriented/CrossProduct.lean

```diff
@@ -0,0 +1,46 @@
+/-
+Copyright (c) 2020 Vedant Gupta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Vedant Gupta, Thomas Browning, Eric Wieser
+-/
+import Mathlib.Analysis.InnerProductSpace.PiL2
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.Basic
+import Mathlib.LinearAlgebra.CrossProduct
+
+/-!
+# Norm of cross-products
+
+This file proves `InnerProductGeometry.norm_withLpEquiv_crossProduct`, relating the norm of the
+cross-product of two real vectors with their individual norms.
+
+## Main definitions
+
+* `InnerProductGeometry.angle` is the undirected angle between two vectors.
+-/
+
+
+assert_not_exists HasFDerivAt ConformalAt
+
+noncomputable section
+
+open Real
+
+open Matrix
+
+namespace InnerProductGeometry
+
+/- The norm of the cross product of two real vectors equals the product of their individual norms
+  times the sine of the angle between them. -/
+theorem norm_withLpEquiv_crossProduct (a b : EuclideanSpace ℝ (Fin 3)) :
+    ‖(WithLp.equiv 2 (Fin 3 → ℝ)).symm (WithLp.equiv _ _ a ×₃ WithLp.equiv _ _ b)‖ =
+    ‖a‖ * ‖b‖ * sin (angle a b) := by
```

Comment:
Can you also add the similar theorem
```lean
theorem norm_withLpEquiv_symm_crossProduct (a b : Fin 3→ ℝ) :
    ‖(WithLp.equiv 2 (Fin 3 → ℝ)).symm (a ×₃ b)‖ =
    ‖WithLp.equiv 2 (Fin 3 → ℝ)).symm a‖ * ‖WithLp.equiv 2 (Fin 3 → ℝ)).symm b‖ *
        sin (angle (WithLp.equiv 2 (Fin 3 → ℝ)).symm a) (WithLp.equiv 2 (Fin 3 → ℝ)).symm b)) := by
```
which should follow immediately from the other one (or vice versa).